src/ProcessManagerBundle/Controller/ExecutableController.php

```diff
@@ -47,7 +47,7 @@ public function runAction(Request $request)
             ]);
         }
 
-        $this->get('process_manager.registry.processes')->get($exe->getType())->run($exe);
+        $this->get('process_manager.registry.processes')->get($exe->getType())->run($exe, $request->request->all());
```

Comment:
I think a form to pre-process the data would be better here -> so we only get validated data. WDYT?

Reply:
done